angular/GettingStarted/getting-started-with-dotnet-cli.md

```diff
@@ -542,7 +542,7 @@ dotnet run
 
 * Open the `.csproj` file which is in project folder and then press `Ctrl+F5` to launch the application in a browser.
 
-![](/angular/GettingStarted/Images/spatemplateoutput.png)
+![angular GettingStarted spatemplateoutput](Images/spatemplateoutput.png)
```

Comment:
Is this correct word? - spatemplateoutput

Reply:
Changed